test/Microsoft.ML.OnnxTransformerTest/Microsoft.ML.OnnxTransformerTest.csproj

```diff
@@ -1,4 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <IsUnitTestProject>true</IsUnitTestProject>
```

Comment:
Can these properties move to the `test\Directory.Build.props` file so they are defined for all test projects?

Reply:
Not every project under test folder is test project (for example TestFrameworkCommon) and these projects that not test project should not run as test project. Other test projects' name are ending with Tests so they are identified as test projects by Arcade and we only need to add these properties into OnnxTransformerTest and TestFramework.

---
In reply to: [539815020](https://github.com/dotnet/machinelearning/pull/5545#discussion_r539815020) [](ancestors = 539815020)